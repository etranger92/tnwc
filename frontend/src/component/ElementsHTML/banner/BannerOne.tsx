/* eslint-disable react/no-unescaped-entities */
import React from "react";
import "./banner.css";

type Props = {
  title: string;
  span: string;
};
const BannerOne: React.FC<Props> = (props) => {
  // eslint-disable-next-line react/prop-types
  const { title, span } = props;
  return (
    <section id="banner-one">
      <article>
        <h1>{title}</h1>
        <span>â€” {span}</span>
      </article>
    </section>
  );
};

export default BannerOne;
