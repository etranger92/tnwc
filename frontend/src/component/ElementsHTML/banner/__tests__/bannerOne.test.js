import * as React from 'react';
import BannerOne from "../BannerOne.tsx";
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe("We test the innerHTML of the BannerOne", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<BannerOne />);
    });
  test("Snapshot BannerOne ", () => {
    //It works
 expect(toJson(wrapper)).toMatchSnapshot();
  });

});

