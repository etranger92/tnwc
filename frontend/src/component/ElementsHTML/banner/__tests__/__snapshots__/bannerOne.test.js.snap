// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`We test the innerHTML of the BannerOne Snapshot BannerOne  1`] = `
<section
  id="banner-one"
>
  <article>
    <h1 />
    <span>
      â€” 
    </span>
  </article>
</section>
`;
